src/core.jsx

```diff
@@ -76,12 +76,12 @@ export default (
     componentWillMount() {
       // Load from hoc argument
       if (isFunction(load)) {
-        load()
+        load(this.props, this.context)
```

Comment:
Yes, there seems to be a bunch of stuff marked as modified in tests that was from before, I think I messed up with git somehow.